script/main.py

```diff
@@ -282,7 +282,31 @@ def users_born(conn, start_date, end_date):
     full_date_of_birth = datetime.strptime(user_date_birth, '%Y-%m-%dT%H:%M:%S.%fZ')
     date_of_birth = full_date_of_birth.date()
     print(f'User {user_name} {user_last_name} was born in {date_of_birth}')
-  
+
+
+def most_secure_password(conn):
+  passwords_list =  [i[2] for i in select_all_gender_and_dob_date_and_password(conn)]  
+  password_and_score_tuple_list = []
+  for password in passwords_list:
+    total = 0
+    if len(password) >= 8:
+      total += 5      
+    if any(letter.isupper() for letter in password):
+      total += 2
+    if any(letter.islower() for letter in password):
+      total += 1
+    if any(letter.isdigit() for letter in password):
+      total += 1
+    if any(not letter.isalnum() for letter in password):
+      total += 3
+
+    password_and_score_tuple_list.append(tuple((password, total)))
+    
+  sorted_by_score = sorted(password_and_score_tuple_list, key=lambda tup: tup[1], reverse=True)
```

Comment:
You don't need to sort if you print all passwords with security >= 7

Sort would be useful to print top N secure passwords